shape_optimization/10_Multi_Constraint_Optimization_3D_Hook/README.md

```diff
@@ -0,0 +1,38 @@
+# Optimization of a Solid Hook
+Optimization of a solid 3D Hook subjected to multiple constraints. 
+
+**Author**: Armin Geiser  
+
+**Kratos version**: 9.0
+
+## Optimization Problem
+- Objective
+    - Minimize mass
+
+- Constraints
+    - constraint 1  &nbsp; : &nbsp; strain energy of main &le; initial value for load case 1
+    - constraint 2  &nbsp; : &nbsp; strain energy of tip &le; initial value for load case 2
+    - constraint 3  &nbsp; : &nbsp; mesh based packaging &le; specified value (use of bounding mesh)  
+
+        <img src="images/3D_Hook_Constraints.svg" height="500">    
+
+- Filter Radius &nbsp; &nbsp; : &nbsp; 25.0
+
+- Mesh motion &nbsp; : &nbsp; YES 
+
+## Optimization algorithm
+- type &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; : &nbsp; gradient_projection
+- number of steps &nbsp;: &nbsp; 20 &nbsp;
+- step size &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; : &nbsp; 3.0
+
+## Results
```

Comment:
I would add two subsections:
- Shape evolution
    - here add the left part of the above image (ideally as a gif) The text overlay is not necessary, we can also describe the content below the image. 
- Convergence Plots